qiskit/transpiler/passes/layout/apply_layout.py

```diff
@@ -94,6 +96,13 @@ def run(self, dag):
                 qargs = [q[new_virtual_to_physical[qarg]] for qarg in node.qargs]
                 new_dag.apply_operation_back(node.op, qargs, node.cargs)
             self.property_set["layout"] = full_layout
+            if (final_layout := self.property_set["final_layout"]) is not None:
```

Comment:
et tu?

![Pacific_Walrus_-_Bull_(8247646168)](https://github.com/Qiskit/qiskit-terra/assets/5968590/0cac5a5b-0093-41bd-8035-3da81820a2ba)

Reply:
Heh, I was wondering if you'd notice, figured I'd give it a try :D

Reply:
I apparently didn't notice it the first time I read it, but saw it when I was checking the changes haha.